src/MongoDB.Bson/ObjectModel/Decimal128.cs

```diff
@@ -830,7 +826,7 @@ public static short ToInt16(Decimal128 d)
                 ulong value;
                 if (Decimal128.TryTruncateToUInt64(d, maxNegativeValue, (ulong)short.MaxValue, out value))
                 {
-                    return Decimal128.IsNegative(d) ? (value == maxNegativeValue ? short.MinValue : (short )(-(short)value)) : (short)value;
+                    return Decimal128.IsNegative(d) ? (value == maxNegativeValue ? short.MinValue : (short)(-(short)value)) : (short)value;
```

Comment:
No changes here, just space removing.

Reply:
Best to revert this then.  It's too hard to see that the change is not substantive.

Reply:
>Best to revert this then. It's too hard to see that the change is not substantive.
Not sure. In vs, there is the built-in format functionality when you click `ctrl +K + D`, all file is being to be formatted at least by some simple rules like unnecessary spaces (that is why I've made this change), so in case if I don't fix this code, all people who will do the same formatting after me will have the same situation.

Reply:
My general rule: minimize diffs.

But if the editor does something automatically there is no need to fight it.